mne/minimum_norm/inverse.py

```diff
@@ -1045,7 +1093,6 @@ def apply_inverse_epochs(epochs, inverse_operator, lambda2, method="dSPM",
     apply_inverse_raw : Apply inverse operator to raw object
     apply_inverse : Apply inverse operator to evoked object
     """
-    _check_reference(epochs)
```

Comment:
wait no, the check was already done at `_apply_inverse_epochs_gen`. This one was redundant.